gems/pending/util/win32/miq-powershell.rb

```diff
@@ -221,9 +225,13 @@ def process_array(node)
     def process_hash(node)
       hsh = {}
       node.each_element do |e|
-        name = convert_type(e.elements[1])
+        # REXML starts its elements indexing at 1, Nokogiri at 0.
```

Comment:
For compatibility XmlHash uses the same indexing as REXML (https://github.com/ManageIQ/manageiq/blob/master/gems/pending/util/xml/xml_hash.rb#L486) but I do not think we need to be concerned with that in the powershell parsing code since we will never use XmlHash here.